src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/RenderMode_Directive_NewExpression/TestComponent.mappings.txt

```diff
@@ -0,0 +1,15 @@
+﻿Source Location: (77:3,0 [137] x:\dir\subdir\Test\TestComponent.cshtml)
+|{

```

Comment:
There's an extra `{` in the source not present in the target, not sure if that's a problem